src/util/rocksdb/mod.rs

```diff
@@ -362,6 +369,7 @@ impl SliceTransform for NoopSliceTransform {
     }
 }
 
+/// Roughly delete files in multiple ranges.
```

Comment:
Can you expand on what it means to "roughly" delete a file? If this corresponds to something obvious that RocksDB users would already know, you don't have to explain it in detail, but maybe indicate what "roughly" means in RocksDB.